kustomize/mariadb-cluster/base/mariadb-galera.yaml

```diff
@@ -123,6 +123,8 @@ spec:
     max_connections=10240
     open_files_limit=10240
     max-connect-errors=1000000
+    wsrep_retry_autocommit=0
+    innodb_lock_wait_timeout=200
```

Comment:
I am curious about this. This value will now make a tranx to wait on a lock for 200sec. This for example would affect select for update queries, but my understanding was that those were all removed from oslo db ? Is that not the case ?